### ISSUE TITLE
Can't install bbb-html5

### ISSUE THREAD

**Issue by jibon57** [bigbluebutton/bigbluebutton#6621]
Recently I am trying to upgrade html5 package but it's showing error like this:

```
The following NEW packages will be installed:
  bbb-html5
0 upgraded, 1 newly installed, 0 to remove and 35 not upgraded.
Need to get 40.3 MB of archives.
After this operation, 105 MB of additional disk space will be used.
Get:1 https://ubuntu.bigbluebutton.org/xenial-200 bigbluebutton-xenial/main amd64 bbb-html5 amd64 1:2.0.0-480 [40.3 MB]
Fetched 40.3 MB in 59s (677 kB/s)                                                                                                           
Selecting previously unselected package bbb-html5.
(Reading database ... 253752 files and directories currently installed.)
Preparing to unpack .../bbb-html5_1%3a2.0.0-480_amd64.deb ...
Unpacking bbb-html5 (1:2.0.0-480) ...
Setting up bbb-html5 (1:2.0.0-480) ...
Installing meteor packages (this may take a moment)
Updating Node packages for bbb-html5 (this may take a moment)
-su: npm: command not found
dpkg: error processing package bbb-html5 (--configure):
 subprocess installed post-installation script returned error exit status 127
Errors were encountered while processing:
 bbb-html5
E: Sub-process /usr/bin/dpkg returned an error code (1)
root@mynaclassroom:/var/cache/apt/archives# npm version
```

but `npm` is present
```
npm -v
6.6.0
```
Please give me suggestion.

**Comment by antobinary** [comment]
Did you follow the 'Step-by-Step Installation' part in http://docs.bigbluebutton.org/html/html5-install.html where we require installation of nodejs and mongo prior to installing bbb-html5?

**Comment by jibon57** [comment]
Yes, I had installed previous version. I tried to get the latest release.

```
dpkg -l | grep nodejs
ii  nodejs                                8.15.0-1nodesource1                        amd64        Node.js event-based server-side javascript engine
```

**Comment by toufikexp** [comment]
Dear ,

   i'm facing the same issue when i'm trying to install html5 , i tried the options to reinstall nodejs but pb still persist

. 
.
.
Created symlink from /etc/systemd/system/default.target.wants/kurento-media-server.service to /usr/lib/systemd/system/kurento-media-server.service.
Setting up bbb-html5 (1:2.2.0-829) ...
cat: /usr/share/etherpad-lite/APIKEY.txt: No such file or directory
**dpkg: error processing package bbb-html5 (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 bbb-html5
E: Sub-process /usr/bin/dpkg returned an error code (1)**



 bbb-conf --status
nginx —————————————————► [✔ - active]
freeswitch ————————————► [✔ - active]
redis-server ——————————► [✔ - active]
bbb-apps-akka —————————► [✔ - active]
bbb-transcode-akka ————► [✔ - active]
bbb-fsesl-akka ————————► [✔ - active]
red5 ——————————————————► [✔ - active]
mongod ————————————————► [✘ - inactive]
bbb-html5 —————————————► [✘ - inactive]
bbb-webrtc-sfu ————————► [✔ - active]
kurento-media-server ——► [✔ - active]
etherpad ——————————————► [✘ - failed]
bbb-web ———————————————► [✔ - active]

**Comment by capilkey** [comment]
@toufikexp You're getting a different error from the original post. When etherpad installs and runs it's supposed to create the APIKEY.txt file, but for some reason that didn't succeed. Can you try manually restarting etherpad "systemctl restart etherpad" and then get the status again.

**Comment by toufikexp** [comment]
Dear ,

for my case i installed  the BBB software in test vm lab , due to several errors occurred   i changed the way how to install it from step by step to the script using "bbb-install.sh" ,  below command : 
wget -qO- https://ubuntu.bigbluebutton.org/bbb-install.sh | bash -s -- -v xenial-220 -g

now my pb , i cannot browse to the greenlight even the installation seems ok 
just to highloght that i'm using test envirenement no DNS and no certificate  configured for 

Regards
Toufik

**Comment by ffdixon** [comment]
I'm not sure we can help much -- your best bet is to install BigBlueButton + Greenlight on an external server with a fully qualified domain name using bbb-install.sh.  This should give you a working setup.

You can then compare the configuration to your internal setup.  Your going to need to get a SSL certificate configured on your internal setup; otherwise, the browsers will prevent users from sharing their webcam/microphone.
